k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientSelectPresenter.kt

```diff
@@ -152,8 +153,11 @@ class RecipientSelectPresenter @Inject constructor(
                     } else {
                         rating
                     }
-                if (isPEpPrivacyProtected && viewRating.value == Rating.pEpRatingHaveNoKey.value
+                if (isPEpPrivacyProtected
                     && view.hasRecipient(recipient)
+                    && ((BuildConfig.IS_ENTERPRISE && PlanckUtils.isRatingUnsecure(viewRating))
```

Comment:
why for the enterprise case we check isRatingUnsecure while for the official build (and all the other cases) we check the rating have no key?

Reply:
This is because for the Play Store version, for the moment we only count unknown recipients.
But in the certified (enterprise) version we need to keep previous behavior, which means that any unsecure recipient is counted.

Reply:
ok clear